src/applications/personalization/profile/components/Profile.jsx

```diff
@@ -165,9 +165,7 @@ class Profile extends Component {
 
     const routes = getRoutes({
       profileContactsPage: toggles.profileContacts,
-      useFieldEditingPage: toggles.profileUseFieldEditingPage,
       profileUseHubPage: toggles.profileUseHubPage,
-      profileShowProofOfVeteranStatus: toggles.profileShowProofOfVeteranStatus,
```

Comment:
@liztownd I noticed `profileShowProofOfVeteranStatus` passed here, but not used in the implementation of `getRoutes` so I removed it. Wanted to confirm that you don't intend to use this anymore.

Reply:
Thanks for checking and sorry! I thought I got them all removed from when we were going to do a whole page, but looks like I missed one. Yeah, that can 100% be removed. Sorry!!